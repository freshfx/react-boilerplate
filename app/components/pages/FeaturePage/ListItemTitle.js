import styled from '@emotion/styled'

const ListItemTitle = styled.p`
  font-weight: bold;
`

export default ListItemTitle
