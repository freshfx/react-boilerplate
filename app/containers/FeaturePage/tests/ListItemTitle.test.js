import 'jest-styled-components'
import React from 'react'
import {mount} from 'enzyme'

import ListItemTitle from '../ListItemTitle'

const renderComponent = (props = {}) => mount(<ListItemTitle {...props} />)

describe('<ListItemTitle />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderComponent()
    expect(renderedComponent).toMatchSnapshot()
  })
})
