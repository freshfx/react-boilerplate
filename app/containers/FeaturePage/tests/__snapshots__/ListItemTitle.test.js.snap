// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItemTitle /> should match snapshot 1`] = `
.c0 {
  font-weight: bold;
}

<ListItemTitle>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "ListItemTitle-sc-1t4skn3-0",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "font-weight:bold;",
          ],
        },
        "displayName": "ListItemTitle",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "ListItemTitle-sc-1t4skn3-0",
        "target": "p",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <p
      className="c0"
    />
  </StyledComponent>
</ListItemTitle>
`;
